build/dependencies.yaml

```
@@ -64,6 +60,14 @@ dependencies:
     - path: test/e2e/framework/nodes_util.go
       match: const etcdImage
 
+  - name: "etcd"
```

maybe give these entries different names?

```suggestion
  - name: "etcd-image"
```

---

Done. Thanks!